### PR TITLE
 added form mui...

### PR DESCRIPTION
**Pull request by BelohvostovAlex**

### CODE REVIEW

File: src/components/Form/Form.jsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import Typography from '@mui/material/Typography';
+import TextField from '@mui/material/TextField';
+import Modal from '@mui/material/Modal';
+import Grid from '@mui/material/Grid';
+
+import { boxStyles, textFieldStyles, typographyStyles } from './styles';
+
+
+export const Form = () => {
+  const [open, setOpen] = React.useState(false);
```
Comment (igorTT) [inline src/components/Form/Form.jsx]: Import useState

File: src/components/Form/styles.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      '& fieldset': {
+        border: 'none',
+      },
+    },
+  };
+};
+
+export const typographyStyles = () => {
+  return { paddingTop: '10px', paddingBottom: '5px' };
+};
```
Comment (igorTT) [inline src/components/Form/styles.js]:
```javascript
export const styles  = {
  box: {
    ...some styles
  },
  textFiled: {
    ...some styles
  },
  etc...
}
```

File: src/components/Form/Form.jsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+import { RoundButton } from '../RoundButton.jsx/RoundButton';
+
+import Container from '@mui/material/Container';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import TextField from '@mui/material/TextField';
+import Modal from '@mui/material/Modal';
+import Grid from '@mui/material/Grid';
+
+import { boxStyles, textFieldStyles, typographyStyles } from './styles';
```
Comment (igorTT) [inline src/components/Form/Form.jsx]:
```javascript
 <Typography sx={styles.textFiled}>
            Title:
          </Typography>

```

File: src/components/MainButton/styles.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+    width: width,
+    height: height ? height : 40,
+    background: color,
+    color: 'white',
+    '&:hover': {
+      background: color,
+    },
+    border: '2px solid black',
+  };
+};
```
Comment (igorTT) [inline src/components/MainButton/styles.js]:
```javascript
export const getStyles= ({ width, height, color }) => ({
  root: {
    width: width,
    height: height ? height : 40,
    background: color,
    color: 'white',
    '&:hover': {
      background: color,
    },
    border: '2px solid black',
  },
};
```

File: src/components/MainButton/MainButton.jsx [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import Button from '@mui/material/Button';
+
+import { buttonStyles } from './styles';
+
+
+export const MainButton = ({ text, color, width, height, onClick }) => {
```
Comment (igorTT) [inline src/components/MainButton/MainButton.jsx]:
```javascript
const classes = getStyles({some props})
return <Button onClick={onClick} sx={classes.root}>{text}</Button>
```

**Review by igorTT — CHANGES_REQUESTED** [review]
Small enhancement

File: src/components/Form/Form.jsx (hunk 1 above)
Comment (BelohvostovAlex) [reply src/components/Form/Form.jsx]: done

File: src/components/Form/styles.js (hunk 2 above)
Comment (BelohvostovAlex) [reply src/components/Form/styles.js]: done

File: src/components/Form/Form.jsx (hunk 3 above)
Comment (BelohvostovAlex) [reply src/components/Form/Form.jsx]: noted, done

File: src/components/MainButton/styles.js (hunk 4 above)
Comment (BelohvostovAlex) [reply src/components/MainButton/styles.js]: noted, changed

File: src/components/MainButton/MainButton.jsx (hunk 5 above)
Comment (BelohvostovAlex) [reply src/components/MainButton/MainButton.jsx]: done

File: src/components/MainButton/styles.js [hunk 6]
```diff
@@ -8,5 +8,22 @@ export const buttonStyles = ({ width, height, color }) => {
...
+
+// export const mergeClasses = (styles1, styles2) => {
+//   if (!styles2) {
+//     return styles1;
+//   }
+//   return {
+//     ...styles1,
+//     ...styles2,
+//   };
+// };
```
Comment (igorTT) [inline src/components/MainButton/styles.js]: Delete it

File: src/components/MainButton/styles.js [hunk 7]
```diff
@@ -8,5 +8,22 @@ export const buttonStyles = ({ width, height, color }) => {
...
+
+export const mergeClasses = (...styles) => {
```
Comment (igorTT) [inline src/components/MainButton/styles.js:14]:
```javascript
export const mergeClasses = (...styles) => {
  return styles.reduce((acc, curr) => ({ ...acc, ...curr }), {});
};
```

**Review by igorTT — CHANGES_REQUESTED** [review]

File: src/components/MainButton/styles.js (hunk 6 above)
Comment (BelohvostovAlex) [reply src/components/MainButton/styles.js]: deleted

File: src/components/MainButton/styles.js (hunk 7 above)
Comment (BelohvostovAlex) [reply src/components/MainButton/styles.js:14]: changed

**Review by igorTT — APPROVED** [review]

**Pull request merged**